README.md

```diff
@@ -0,0 +1,14 @@
+# tf-mod-azure-mgmt-lock
```

Comment:
This README is still lackluster, ref. my comment here: https://github.com/dsb-norge/tf-mod-azure-mgmt-lock/pull/1#discussion_r1022833597